heareval/predictions/runner.py

```diff
@@ -74,6 +82,21 @@ def runner(
         if not task_path.is_dir():
             raise ValueError(f"{task_path} should be a directory")
 
+        # Get the output directory name
+        output_path: Path
+        if outdir is None:
+            # Default is a folder predictions/embedding_name/task_name
+            output_path = Path("predictions").joinpath(task_path.parent.name)
+            output_path = output_path.joinpath(task_path.name)
+        else:
+            # A separate output directory was passed in.
+            # Create a subdirectory within that with the same name as
+            # the task to save the results in.
+            output_path = Path(outdir).joinpath(task_path.name)
```

Comment:
In embeddings, if the embedding dir exists but is incomplete, we remove it, just so as not to have mixed results. We should do that here too. (Unless it's the embedding dir. TBH I think embedding dir should never equal predictions dir.)

Reply:
Updated to remove the predictions dir if it is incomplete. Also added assertion to verify it is not the same as the input.